### DISCUSSION TITLE
Github Achievement hasn't  come still

### DISCUSSION THREAD

**Discussion by moozunch** [community/community#118288] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hi, so I just got two accepted answear, one which is in march 11th and the second one is today 2 hours ago exactly which if I read correctlly I should already got Galaxy Brain achivement but I hasn't got the achievement yet, is there any rules to get them like wait a bit more or is it a bug which I should report it? if so let me know where to report it, Thankyou!

**Comment by Duk3Dum0ntF4n** [comment, marked as answer]
Here github described their changes in community achievement:
https://github.com/orgs/community/discussions/106536

If to summarize there were a lot of low quality and spam discussions, so now it is a bit difficult to get it
